charts/astronomer/tests/houston_pdb_test.yaml

```diff
@@ -0,0 +1,46 @@
+suite: Test houston pod disruption budget
+# if omitted, all templates are rendered
+# templates:
+tests:
+- it: should have the PDB labels on the API deployment
+  asserts:
+    - equal:
+        path: spec.template.metadata.labels.tier
+        value: astronomer
+      template: houston/api/houston-deployment.yaml
+    - equal:
+        path: spec.template.metadata.labels.component
+        value: houston
+      template: houston/api/houston-deployment.yaml
+- it: PDB labels match the deployment
+  asserts:
+    - equal:
+        path: spec.selector.matchLabels.tier
+        value: astronomer
+      template: houston/api/houston-pod-disruption-budget.yaml
+    - equal:
+        path: spec.selector.matchLabels.component
+        value: houston
+      template: houston/api/houston-pod-disruption-budget.yaml
+- it: does not match cronjobs or workers
+  asserts:
+    - notEqual:
+        path: spec.jobTemplate.spec.template.metadata.labels.component
+        value: houston
+      template: houston/cronjobs/houston-expire-deployments-cronjob.yaml
+    - notEqual:
+        path: spec.jobTemplate.spec.template.metadata.labels.component
+        value: houston
+      template: houston/cronjobs/houston-check-updates-cronjob.yaml
+    - notEqual:
+        path: spec.jobTemplate.spec.template.metadata.labels.component
+        value: houston
+      template: houston/cronjobs/houston-cleanup-deployments-cronjob.yaml
+    - notEqual:
+        path: spec.jobTemplate.spec.template.metadata.labels.component
+        value: houston
+      template: houston/cronjobs/houston-check-updates-cronjob.yaml
+    - notEqual:
+        path: spec.selector.matchLabels.component
+        value: houston
+      template: houston/worker/houston-worker-deployment.yaml
```

Comment:
it would be nice if I could accomplish "no other pods have the labels", but I don't think this kind of logic can be done, instead it's only best at asserting directly with 'full path' you might call it